docker/images/docker-spark/master/master.sh

```diff
@@ -26,7 +26,7 @@ if curl --fail -X POST --output /dev/null --silent --head http://127.0.0.1:8080;
 fi
 
 ## Supervisor will monitor the ingestion to ensure it stays up
-supervisord -c /supervisord.conf
+service supervisor start &
```

Comment:
instead of doing this can't you start supervisord and have supervisord manage all of the other processes below?

Reply:
great point! I'll work on that and let you know when it's updated